main/writable_nested_serializers.py

```diff
@@ -0,0 +1,425 @@
+# -*- coding: utf-8 -*-
```

Comment:
Is this copied from deep or drf_writable_nested?
if it's drf_writable_nested then we can just add this to requirement.txt else fine.

Reply:
Copied from `deep`